django/conf/global_settings.py

```diff
@@ -223,6 +223,9 @@ def gettext_noop(s):
 # Set to True to assume "https" during the Django 5.x release cycle.
 FORMS_URLFIELD_ASSUME_HTTPS = False
 
+# ToDo: Documentation string
+FORMS_DEFAULT_BLANK_CHOICE_LABEL = gettext_noop("- Select an option -")
```

Comment:
I decided to create a setting for the label rather than the original list-with-tuple as in `BLANK_CHOICE_DASH`. This makes it simpler for developers to override. What do you think about this?

Reply:
That seems OK. (🤷) It's hard to think of how it might affect usage. (≈No-one is overriding this already I think.) 

Not sure about the name 🤔

Reply:
I dislike the name, so any shorter suggestions are welcome!

NB. The original name was only "BLANK_CHOICE", but the new value is only a label and not a complete choice.